spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -875,8 +876,8 @@ def _run(self, run_time, run_until_complete=False):
                     break
 
         # Disable auto pause and resume if the binary can't do it
-        if (self._executable_start_type !=
-                ExecutableStartType.USES_SIMULATION_INTERFACE):
+        if (ExecutableType.USES_SIMULATION_INTERFACE not in
+                self._executable_types):
```

Comment:
you can. the other versions are not considered in the simulation calls. and for running, it is always in running state, so thats fine. sync is a bit intereting, as its meant to go to exit before auto pause and resume kicks in. so itll stay in exit state, as the runnign state is also allowing exit.

Reply:
Hmm, not sure I like this then as those cores won't pause even though the simulation has paused.  I guess this depends on the semantics of those cores in that case...

Reply:
well in the cases we have. not pausing is exactly what we want them to do. extra monitor cores needs to be runnign for it to do the data speed up.

Reply:
Hmm yes, I can see why you need this here - still not sure I like it!  Basically you are saying that the only cores that are allowed to do this are those that we add.  In that case, maybe we should have a state that is something like ExecutableType.SYSTEM.  This would split it out from those cores that will reach exit state, or some intermediate sync state.

Reply:
Im inclined to not do this until we find a use case. We could try anticipating the needs of invisible things. but unless theres a valid usecase, i think its wasted cycles.

Reply:
I can see this going wrong quickly - again the sticky plaster solution.  We need to say what we mean - we already have two separate executable types here being specified as one (something that starts and ends vs. something that starts and never ends).

Reply:
fixed me thinks